Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM alpine:3.7
-RUN apk add --no-cache ca-certificates
-RUN adduser -D -g '' vault-manager
-USER vault-manager
-COPY vault-manager query.graphql /
+FROM registry.access.redhat.com/ubi8/go-toolset:latest as builder
```

Comment:
Can we use a version here instead of latest?

Reply:
yes we should! fixed!